components/eamxx/cmake/machine-files/pm-cpu.cmake

```diff
@@ -5,6 +5,9 @@ include (${EKAT_MACH_FILES_PATH}/kokkos/amd-zen3.cmake)
 include (${EKAT_MACH_FILES_PATH}/kokkos/openmp.cmake)
 
 set(CMAKE_CXX_FLAGS "-DTHRUST_IGNORE_CUB_VERSION_CHECK" CACHE STRING "" FORCE)
+set(PYBIND11_PYTHON_VERSION 3.9 CACHE STRING "")
```

Comment:
@elynnwu or @frodre , I think there is consensus that we can merge this in as long as we put guard rails on the environment call.  Can you add a compiler flag to turn these options "OFF" by default and only "ON" when the user specifies they want ML?  I would be in favor of making the flag explicit in name, like `FV3NET` or `CorrectiveML`.  You would want to add the flags to both gpu and cpu config files.

@mahf708 have I characterized what needs to be done correctly?